src/main/java/net/coderbot/iris/gl/blending/AlphaTest.java

```diff
@@ -16,15 +18,19 @@ public String toExpression(String indentation) {
 	}
 
 	public String toExpression(String alphaAccessor, String indentation) {
+		String expr = function.getExpression();
```

Comment:
This isn't very neat but this function is a mess anyways and will probably get yeeted with glsl-transformer, so I guess it goes for now.